clang/test/CodeGenHLSL/intrinsic.ll

```diff
@@ -0,0 +1,19 @@
+; Function Attrs: alwaysinline mustprogress nofree norecurse nosync nounwind willreturn memory(none)
```

Comment:
Can you help me understand what this file is for please?  It doesn't look like it is actually testing anything.

Reply:
In retrospect, it looks like this file is an accidental commit.  Please delete it. you can ignore my other comments on this file.